test/expected/build/library.j/dest/resources/library/j/ES6.features.2.js

```diff
@@ -21,14 +21,13 @@ window.someRandomModule ||
 		 * @version ${version}
 		 *
 		 * @constructor
-		 * @extends library.j.a
+		 * @extends library.j.Control
 		 * @public
 		 * @since 1.22
-		 * @alias library.j.aaa
-		 * @ui5-metamodel This control will also be described in the UI5 (legacy) design time meta model.
+		 * @alias library.j.SubControl
 		 */
-		(a) =>
-			a.extend(`library.j.aaa`, {
+		(Control) =>
+			Control.extend(`library.j.SubControl`, {
```

Comment:
Still, the control name doesn't fit to the name of the file / module (`library/j/SubControl` vs `library/j/ES6.features.2`).

Reply:
Done